regression-test/suites/ddl_p0/test_create_table_auto_partition.groovy

```diff
@@ -22,9 +22,9 @@
 suite("test_create_table_auto_partition") {
     def testTable = "test_create_table_auto_partition_table"
 
-    sql "DROP TABLE IF EXISTS ${testTable}"
+    sql "DROP TABLE IF EXISTS test_create_table_auto_partition_table"
```

Comment:
Unexpected changes?

Reply:
all table name in testcases should avoid use variable. So I think this is proper

Reply:
so should remove `def testTable = "test_create_table_auto_partition_table"`?

Reply:
yes. fixed. thx~